fusor-ember-cli/app/components/cancel-deployment-modal.js

```diff
@@ -2,6 +2,16 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
 
+  title: Ember.computed('deploymentName', function() {
```

Comment:
Also not sure why it was removed.
On Jul 14, 2016 8:56 PM, "Erik Nelson" <notifications@github.com> wrote:

> In fusor-ember-cli/app/components/cancel-deployment-modal.js
> <https://github.com/fusor/fusor/pull/1040#discussion_r70852188>:
>
> > @@ -2,6 +2,16 @@ import Ember from 'ember';
> >
> >  export default Ember.Component.extend({
> >
> > +  title: Ember.computed('deploymentName', function() {
>
> Not sure, I don't know of any reason.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/fusor/fusor/pull/1040/files/044696019c5d1d35162cadfd928102c111e18031#r70852188>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAe2Mnr2uL65rgHXvC6tU5wREKJdqkTaks5qVnhMgaJpZM4JMl2L>
> .
>